Helper/Url.php

```diff
@@ -278,6 +279,16 @@ public function getPreviewUrlSearch(Store $store)
         return $this->addNostoDebugParamToUrl($url);
     }
 
+    /**
+     * Returns the store domain
+     *
+     * @return string
+     */
+    public function getActiveDomain(Store $store)
+    {
```

Comment:
We should prolly catch the possible exception from `UrlHelper::parseDomain` and just return empty string. Otherwise the API calls are not made if the URL parsing fails